### ISSUE TITLE
Uninitialized constant using Rails 4.2

### ISSUE THREAD

**Issue by lbstephane** [alexreisner/geocoder#834]
Hi,

I'm use different fields for my address in database. So I need to concat LibAddr, LibZipc, LibCity and LibCtry.

Using this return an uninitialized constant : 

geocoded_by :full_address
after_validation :geocode
	
def full_address
	[LibAddr, LibZipc ,Libcity, LibCtry].compact.join(', ')
end

Trying :LibAddr and so on return nothing and trying only geocoded_by :LibAddr work fine.

Precision : Informations are posted to the controller as a json post (API)

Thanks

Update : Trying with a view form in another project work, so, what's the method for an api. Thanks.

**Comment by lbstephane** [comment]
Hi (again lol)

I found a way but i'm not sure it is the best. So I added a LibFullAddr param in my database.

Now i use geocoded_by :LibFullAddr

And in my controller I merge  LibFullAddr in posted params using 

def set_full_address
     [params[:site][:LibAddr], params[:site][:LibZipc] ,params[:site][:LibCity], params[:site [:LibCtry]].compact.join(', ')
end

It seems to work fine.

Let me know if there is a simple way when using json post, patch or put

Best regards.
